frontend/src/components/games/list/GameListItem.tsx

```diff
@@ -153,6 +235,38 @@ export function RenderResult(params: GridRenderCellParams) {
     );
 }
 
+function parseTimeControl(
+    tc: string | undefined,
+): [number | undefined, number | undefined] {
+    if (!tc) {
+        return [undefined, undefined];
+    }
+
+    const [gameTime, increment] = tc
+        .split(/[+|\s]+/)
+        .flatMap((v) => (v ? Number(v) : []));
+
+    if (!gameTime) {
+        return [undefined, undefined];
+    }
+
+    return [Math.round(gameTime / 60), increment];
+}
+
+export function getTimeControl({ timeControl }: { timeControl?: string }) {
+    const [gameTime, increment] = parseTimeControl(timeControl);
+
+    return gameTime ? `${gameTime}+${increment ?? 0}` : '';
+}
+
+export function RenderTimeControl({ timeControl }: { timeControl?: string }) {
```

Comment:
Should this return null if timeControl is undefined/empty rather than rendering an empty div/p?